config/env.go

```diff
@@ -0,0 +1,131 @@
+package config
+
+import (
+	"crypto/tls"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// The following variables contain the names of environment variables that can
```

Comment:
Just noticed this small typo:

```suggestion
// The following constants contain the names of environment variables that can
```